servers/rendering/renderer_rd/shaders/effects/fsr2_accumulate_pass.glsl

```diff
@@ -0,0 +1,8 @@
+#[compute]
+
+#version 450
+
+#VERSION_DEFINES
+
+#include "motion_vector_inc.glsl"
+#include "thirdparty/amd-fsr2/shaders/ffx_fsr2_accumulate_pass.glsl"
```

Comment:
nitpicking, needs a newline at the end, same goes for a number of the other files :)

Reply:
Sounds good to me.